core/src/commands/index.ts

```diff
@@ -99,10 +104,14 @@ const getCommands: () => ICommand[] = () => [
   codeBlock,
   comment,
   image,
+  table,
   divider,
   unorderedListCommand,
   orderedListCommand,
   checkedListCommand,
+  divider,
+  issue,
```

Comment:
@RARgames I think it is not perfect and should not be placed in the default `commands`.